content/en/docs/advanced-tasks/fail-forward-upgrades.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Opting into fail forward upgrades"
+linkTitle: "Opting into fail forward upgrades"
+weight: 4
+description: >
+  Fail forward upgrades is a feature that allows for the automatic upgrade of failed installs and upgrades. 
+  It is ultimately unsafe and should only be used in specific circumstances. 
+---
+
+## Warning
+
+Enabling fail forward upgrades may have severe consequences and recovery from these consequences is not guaranteed. Critical versions of operators may fail to install and can thus lead to unknown behavior and potentially data loss. As a result, using this features is not recommended in the majority of circumstances. Only use this feature if you:
+
+- Have complete control over operator installs, including OLM.
+- Do not want to manually upgrade your failed operators.
+
+## What "Fail Forward" upgrades add
+A failed installation/upgrade is typically caused by one of two scenarios:
+
+- **A Failed CSV:** The CSV is in the FAILED phase.
```

Comment:
Do we need to define CSV here?

Reply:
How about a little introduction and then a link to the documentation for it?

https://olm.operatorframework.io/docs/concepts/crds/clusterserviceversion/#docsconcepts